src/client/src/index.css

```diff
@@ -157,3 +157,40 @@
 .Mui-checked {
   color: black !important;
 }
+
+
+/*
+* Styles previously set by Semantic UI Css
+ */
+
```

Comment:
REQ: Kommentar umschreiben oder löschen, da man in Zukunft (hoffentlich) nicht mehr wissen muss, dass Semantic UI mal drin war :-)